plasmapy/particles/atomic.py

```diff
@@ -664,11 +664,11 @@ def common_isotopes_for_element(
         CommonIsotopes = [
             isotope
             for isotope in isotopes
-            if "abundance" in _isotopes.data_about_isotopes[isotope]
+            if "abundance" in _isotopes.data_about_isotopes[isotope.isotope]
```

Comment:
`Particle.isotope` returns a string with the symbol of the isotope, which is expected by that `dict`.

Reply:
I thought we didn't return dict objects?

Reply:
The dict here was a private one containing isotope data, so it wasn't part of our public API.  I'd say that there are times when a dict is a good choice for a return value, and other times when it's worth creating something like a class or namedtuple.  As with everything, there are tradeoffs!